app/controllers/pools_controller.rb

```diff
@@ -2,8 +2,8 @@ class PoolsController < ApplicationController
   before_action :ensure_logged_in, only: [:create, :destroy]
 
   def create
-    name = "Pool #{Tournament.find(pool_params[:tournament_id]).pools.count + 1}"
-    @pool = Pool.create(name: name, tournament_id: pool_params[:tournament_id])
+    @pool = Pool.create(name: 'New pool', tournament_id: pool_params[:tournament_id])
```

Comment:
Instead of `create`, you can call `new`, which makes the ActiveRecord object in Ruby, but doesn't write to the database until you call `save` a couple of lines below.